documents/policies/external/Competition Requirements.md

```diff
@@ -119,6 +138,11 @@ The WCA Delegate must submit the following information when requesting approval
    2. Only results from events that appear on the competition website in the Events tab and Schedule will be recognized.
       1. Other events may only be held unofficially.
    3. If the main event of the competition is not the 3x3x3 Cube, it should be publicly mentioned in the competition's information.
+      1. The main event should be the main focus of the competition by being recognizable in the competition schedule and having an appropriate proportion of rounds and attempts.
```

Comment:
What does this mean? I would argue that the "main focus" of a competition is having fun and meeting friends. As an organiser I normally announce the winner of the main event as the winner of the competition and make a point of it at awards, which I feel is sufficient but I'm not sure would meet this threshold.

At "standard" competitions 3x3 is just the last event held with the most competitors and most rounds - it could be a main/not a focus/one of many focuses depending on the person. As a different example, for a competition the "main focus" could be "side-events" but you pick e.g. Skewb because it has the last final held and has the equal most rounds, but that doesn't make it the "main focus", although it could be the "main event". For "PBQ" competitions FMC could be the main event, but it is hard make it recognisable given the nature of the event and competition logistics requirements, however it can be made worthy of being a main event during the presentation of awards/prizes, for example.

Reply:
I agree with Ed. I would propose the following instead:

The main event should be ~~the main focus of the competition by being~~ recognizable in the competition schedule and having an appropriate proportion of rounds and attempts.

Reply:
Personally, advancements and cutoffs play a big part in how I see a main event as well. So what Sébastien wrote seems good to me.